res/openvpn-gui-res-it.rc

```diff
@@ -410,6 +410,8 @@ Per maggiori dettagli consulta il file log (%ls) ."
     IDS_NFO_SERVICE_ACTIVE_EXIT "Al momento sei connesso (il servizio di OpenVPN è in funzione). \
 Se esci dall'interfaccia di OpenVPN rimarrai comunque connesso.\n\n\
 Vuoi continuare e uscire dall'interfaccia di OpenVPN?"
+    IDS_NFO_STATE_ROUTE_ERROR "Stato corrente: connesso con errori di rotta"
```

Comment:
Maybe better "con errori nelle rotte". It is basically a compressed form or "errori durante l'installazione delle rotte"